source/ota.c

```diff
@@ -3170,7 +3215,19 @@ OtaErr_t OTA_Init( OtaAppBuffer_t * pOtaBuffer,
         /*
          * Initialize OTA event interface.
          */
-        ( void ) otaAgent.pOtaInterface->os.event.init( NULL );
+        if( !otaAgent.isOtaInterfaceInited )
+        {
+            ( void ) otaAgent.pOtaInterface->os.event.init( NULL );
+            otaAgent.isOtaInterfaceInited = 1;
+        }
+        else
+        {
+            /* Drop all events that created after OTA_Shutdown. */
+            while( otaAgent.pOtaInterface->os.event.recv( NULL, &eventMsg, 0 ) == OtaOsSuccess )
+            {
+                LogDebug( ( "Event(%d) is dropped.\r\n", eventMsg.eventId ) );
```

Comment:
This can be a warning log to indicate the event ID dropped.

Reply:
Fixed in latest commit.